pkg/installmanager/installmanager.go

```diff
@@ -1810,3 +1820,62 @@ func (m *InstallManager) overrideCredentialsModeOnInstallConfig(credentialsMode
 
 	return nil
 }
+
+func (m *InstallManager) patchAzureOverrideCreds() error {
+	overrideCredsPath := filepath.Join(m.WorkDir, "manifests", overrideCredsYAML)
+	overrideCredsBytes, err := ioutil.ReadFile(overrideCredsPath)
+	if err != nil {
```

Comment:
We need a check for file existence somewhere in here, right? Or is the file definitely supposed to exist by this point (whether generated by installer or provided by the user)?

Reply:
Wait wait. I know we designed it so the two things would be used together for this use case, but empirically they're not actually related. So I still think we should add a file existence check and _not_ fail if the file isn't provided.

Reply:
OK. I changed the logic, if the 99_cloud-creds-secret.yaml doesn't exist then go ahead for install